### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by dfridovi** [dfridovi/mininet#2]
Use a validation set to measure loss at the end of each epoch, rather than just the training set.

**Comment by dfridovi** [comment]
Done.
